automatic/PDFCreator/PDFCreator.nuspec

```diff
@@ -14,11 +14,9 @@
 PDFCreator – PDFCreator easily creates PDFs from any Windows program. Use it like a printer in Word, StarCalc or any other Windows application.
     </description>
     <summary>PDFCreator - PDFCreator easily creates PDFs from any Windows program.</summary>
-    <tags>pdf admin</tags>
-    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages</packageSourceUrl>
+    <tags>pdf editor printer admin</tags>
```

Comment:
Should be tagged with [fos](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guideliness-%5BDraft%5D#122-obligatory-tags).